packages/pagination/README.md

```diff
@@ -0,0 +1,15 @@
+# Pagination
+
+A `Pagination` component may be used to navigate through a large colletion of items that is divided into multiple pages.
+
+## Layout
+
+The `Pagination` component should be right-aligned with the list of items it is paging through.
+
+## [WIP] Customizing item count per page
+
+TODO: Work with design team to spec how we use the "items per page" dropdown menu
```

Comment:
@mperrotti  As we discussed in the meeting, please use the sentences case for the label:
"items per page" change to "Items per page".

And one thing we forgot to mentioned, the default page input value  should be 30. Not sure we have this default number in this PR. Please check. Thank u, Mike ;)

Reply:
> "items per page" change to "Items per page"

Done

> the default page input value should be 30

Was already done :)

Reply:
I also updated the documentation with the info you and Schwartz added in Invision